spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingServiceProperties.java

```diff
@@ -50,6 +50,8 @@
 
 	private String defaultBinder;
 
+	private String[] outboundChannelNames = new String[0];
```

Comment:
Also we're not strictly talking about "channels" here, but "destinations". Perhaps the property could be renamed something like `routableDestinations`?